src/material-examples/focus-monitor-directives/focus-monitor-directives-example.ts

```diff
@@ -8,8 +8,8 @@ import {ChangeDetectorRef, Component, NgZone} from '@angular/core';
   styleUrls: ['focus-monitor-directives-example.css']
 })
 export class FocusMonitorDirectivesExample {
-  elementOrigin: string = this.formatOrigin(null);
-  subtreeOrigin: string = this.formatOrigin(null);
+  elementOrigin = this.formatOrigin(null);
```

Comment:
Why was the type removed here?

Reply:
Well, `formatOrigin` returns a `string`, so `elementOrigin` is automatically inferred as `string`, there's no necessity to be redundant.